tests/scripts/basic-build-test.sh

```diff
@@ -42,6 +42,7 @@ fi
 : ${GNUTLS_SERV:="gnutls-serv"}
 : ${GNUTLS_LEGACY_CLI:="$GNUTLS_CLI"}
 : ${GNUTLS_LEGACY_SERV:="$GNUTLS_SERV"}
+: ${SEED:=1}
```

Comment:
What does the seed have to do with adding test coverage threshold testing? Seed related changes should go in a separate commit.

Reply:
Fair point. I've split this into another commit, although doing this doesn't seem very common in the project.